tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -158,6 +164,29 @@ describe(__filename, () => {
       expect(reviewCountLink.children()).toHaveText(ratingsCount.toString());
     });
 
+    it('renders links with `src` query parameters when the location has one', () => {
+      const slug = 'some-slug';
+      const src = 'some-src';
+      const location = createFakeLocation({ query: { src } });
+
+      const root = render({
+        addon: createInternalAddon({
+          ...fakeAddon,
+          ratings: { text_count: 3, count: 123 },
+          slug,
+        }),
+        location,
+      });
+
+      const reviewTitleLink = getReviewTitle(root).find(Link);
+      const reviewCountLink = getReviewCount(root).find(Link);
+
+      const listURL = reviewListURL({ addonSlug: slug, src });
+
+      expect(reviewTitleLink).toHaveProp('to', listURL);
+      expect(reviewCountLink).toHaveProp('to', listURL);
+    });
+
```

Comment:
do we need a test that checks when src isn't set (or is `''`) or would that be covered by other existing tests?

Reply:
it is sort of covered by the other test cases because either we have a `src` or we don't (existing behavior). We don't do anything particular when the `src` has an empty value. It's fairly harmless.